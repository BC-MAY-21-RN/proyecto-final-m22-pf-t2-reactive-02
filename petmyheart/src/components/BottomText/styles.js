import {StyleSheet} from 'react-native';

const styles = StyleSheet.create({
  container: {marginTop: 20, alignSelf: 'center'},
  text: {
    fontSize: 16,
    textDecorationLine: 'underline',
    fontStyle: 'italic',
  },
});

export default styles;
