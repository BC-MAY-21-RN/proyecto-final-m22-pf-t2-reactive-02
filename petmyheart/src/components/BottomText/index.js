import React from 'react';
import {Text, TouchableOpacity} from 'react-native';
import styles from './styles';

export default function BottomText({text}) {
  return (
    <TouchableOpacity style={styles.container}>
      <Text style={styles.text}>{text}</Text>
    </TouchableOpacity>
  );
}
