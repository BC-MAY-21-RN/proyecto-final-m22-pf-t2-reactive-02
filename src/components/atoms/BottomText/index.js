import React from 'react';
import {Text, TouchableOpacity} from 'react-native';
import styles from './styles';

export default function BottomText({text, navigation, namescreen}) {
  return (
    <TouchableOpacity
      onPress={() => navigation.navigate(namescreen)}
      style={styles.container}>
      <Text style={styles.text}>{text}</Text>
    </TouchableOpacity>
  );
}
