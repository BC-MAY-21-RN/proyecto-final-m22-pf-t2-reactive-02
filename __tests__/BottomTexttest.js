import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import BottomText from '../src/components/atoms/BottomText';

describe('Test from BottomText component', () => {
  it('BottomText snapshot test', () => {
    const renderBottomText = renderer
      .create(
        <BottomText
          namescreen={''}
          navigation={{navigation: () => {}}}
          text={''}
          key={1}
        />,
      )
      .toJSON();
    expect(renderBottomText).toMatchSnapshot();
  });
});
