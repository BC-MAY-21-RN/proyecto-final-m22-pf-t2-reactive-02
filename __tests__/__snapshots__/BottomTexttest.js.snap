// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test from BottomText component BottomText snapshot test 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignSelf": "center",
      "marginTop": 20,
      "opacity": 1,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 16,
        "fontStyle": "italic",
        "textDecorationLine": "underline",
      }
    }
  />
</View>
`;
